### DISCUSSION TITLE
Docs example unclear

### DISCUSSION THREAD

**Discussion by avieini-sunbit** [cooper-org/cooper#69] [Q&A]
Hi all , 
I run into this wonderful package and I want to create a FPR@PPR classifier .
If I follow the example in the docs (which is constraint over PPR ):
https://cooper.readthedocs.io/en/latest/auto_tutorials/plot_gaussian_mixture.html#sphx-glr-auto-tutorials-plot-gaussian-mixture-py
in line 122-123 we construct our constraint which based on hinge loss .
The example use the following : 
```
`             # Hinge approximation of the rate
             probs = torch.sigmoid(logits)
             hinge = torch.mean(torch.max(torch.zeros_like(probs), 1 - probs))

             # level - proxy_ineq_defect <= 0
             hinge_defect = self.const_level - hinge
```
1. why using probs instead of logits?
2. why calculating overall hinge loss instead calculating tp+fp/|n| which is PPR)? 
I think I'm missing something and I would be glad if you can help me out on that 

Thanks in advance 
Avi
